src/index.hbs

```diff
@@ -9,6 +9,12 @@
 	<script src="https://static.amsterdam.nl/fonts/mtiFontTrackingCode.min.js"></script>
 	<link href="https://static.amsterdam.nl/fonts/fonts.css" rel="stylesheet" />
 	<title>DMS Upload</title>
+	<style>
+		/* Ensure that the react-select dropdown is on top in case it is present inside a modal */
+		body > div:last-child {
```

Comment:
not better to address it specificly in the style of the creatable by id? as now we have no way of knowing that the last div "always" contains the creatable. In the future we can add components, which also have a div on the end of the page.

Reply:
Solved it from the bmi-component-library side via a prop